enumerable_method.rb

```diff
@@ -0,0 +1,123 @@
+module Enumerable
+
+    def my_each 
+      i=0
```

Comment:
Make sure that code is clean: indentation with two spaces, line lengths, spaces around operators - first section of [this link](https://github.com/github/rubocop-github/blob/master/STYLEGUIDE.md#ruby-style-guide).